src/cljc/quil/core.cljc

```diff
@@ -4367,18 +4366,18 @@
              (float x2) (float y2)
              (float x3) (float y3)))
 
-(defn
-  ^{:require-binding false
-    :processing-name "unbinary()"
-    :category "Data"
-    :subcategory "Conversion"
-    :added "1.0"}
-  unbinary
-  "Unpack a binary string to an integer. See binary for converting
-  integers to strings."
-  [str-val]
-  #?(:clj (PApplet/unbinary (str str-val))
-     :cljs (.unbinary (ap/current-applet) (str str-val))))
```

Comment:
Same here, can use `(js/parseInt str-val 2)` to parse from binary.

Reply:
👍